int32 linear_pos
int32 servo_pos
---
bool done
